src/elements/content-sidebar/activity-feed/activity-feed/ActivityThread.scss

```diff
@@ -0,0 +1,32 @@
+@import '../../../common/variables';
+
+.bcs-ActivityThread {
+    &-reply.bcs-ActivityItem {
```

Comment:
We've typically avoided this notation so that searching the codebase works more effectively